src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -1,3 +1,7 @@
+import { ConfiguredDocumentClass } from '../../../types/helperTypes';
+import { DocumentModificationOptions } from '../../common/abstract/document.mjs';
+import { BaseActiveEffect, BaseActor, BaseItem, BaseToken, BaseUser } from '../../common/documents.mjs';
```

Comment:
Instead of importing these, we can just use them from the global `foundry.documents`. That way, it's a bit more in sync to how things are done in foundry.js (which doesn't import anything, as it's a script).